core/src/main/kotlin/org/evomaster/core/search/gene/SqlJSONGene.kt

```diff
@@ -0,0 +1,47 @@
+package org.evomaster.core.search.gene
+
+import org.evomaster.core.output.OutputFormat
+import org.evomaster.core.search.service.Randomness
+
+class SqlJSONGene(name: String, val objectGene: ObjectGene = ObjectGene(name, fields = listOf())) : Gene(name) {
+
+
+    override fun copy(): Gene = SqlJSONGene(
+            name,
+            objectGene = this.objectGene.copy() as ObjectGene)
+
+
+    override fun randomize(randomness: Randomness, forceNewValue: Boolean, allGenes: List<Gene>) {
+        objectGene.randomize(randomness, forceNewValue, allGenes)
+    }
+
+
+    override fun getValueAsPrintableString(previousGenes: List<Gene>, mode: String?, targetFormat: OutputFormat?): String {
+        return "\"%s\"".format(objectGene.getValueAsPrintableString(previousGenes, mode, targetFormat))
+    }
```

Comment:
can u comment this `"\"%s\""`?

Reply:
I cloned the same code (with a proper TODO comment for future refactor)  than StringGene.getValueAsPrintableString()